### ISSUE TITLE
Old documentation links are 404

### ISSUE THREAD

**Issue by jni** [napari/napari#1702]
## 🐛 Bug

Our transition to versioned docs has meant that many (all?) old links to the docs are 404. (e.g. https://napari.org/docs/developers/GOVERNANCE.html). Currently Google is still finding them. They should instead redirect to or contain a copy of dev, imho. (or latest stable release.)

**Comment by sofroniewn** [comment]
Good catch! I'd probably link to `dev` as that will be stable moving forward

**Comment by sofroniewn** [comment]
i.e. https://napari.org/docs/developers/GOVERNANCE.html should redirect to https://napari.org/docs/dev/developers/GOVERNANCE.html

BUT https://napari.org/docs/0.3.8/developers/GOVERNANCE.html should stay as is

**Comment by sofroniewn** [comment]
This was closed by #1750 that fixed the main links in our readme by adding `dev`. There had been some discussion at one point with @jni about trying to do redirects from old links (see https://github.com/napari/napari/issues/1702#issuecomment-709616237) but I think we're probably ok now. Google searches have started surfacing real links again. While we've got more website reorganization ahead, we'll likely keep this main url naming convention the same, so fine to leave closed.
